CemrgApp/Applications/MainApp/target_libraries.cmake

```diff
@@ -7,7 +7,7 @@ set(target_libraries
 
   org_mitk_gui_qt_datamanager
   org_mitk_gui_qt_basicimageprocessing
-  org_mitk_gui_qt_dicom
```

Comment:
What was the difference between dicom and dicombrowser?

Reply:
It was renamed in v2021.02, see https://phabricator.mitk.org/T27001